src/main/scala/models/Connector.scala

```diff
@@ -7,9 +7,11 @@ import io.circe.generic.semiauto.*
 
 import java.util.UUID
 
-case class Namespace(
+case class Connector(
   id: UUID,
   appId: UUID,
   name: String,
+  typ: Option[String],
+  config: Option[String],
```

Comment:
Maybe add additional field for sensitive data (auth tokens, credentials), so it can be treated differently?